cmd/podman/root.go

```diff
@@ -474,12 +474,6 @@ func resolveDestination() (string, string, string) {
 		return "", uri, ident
 	}
 
-	cfg, err := config.ReadCustomConfig()
-	if err != nil {
-		logrus.Warning(errors.Wrap(err, "unable to read local containers.conf"))
-		return "", registry.DefaultAPIAddress(), ""
-	}
-
```

Comment:
This doesn't look correct, we explicitly want only the user config here for the remote connections. Now it will use the full config which means that rootless users could end up with connections made by root. @jwhonce agree?

Reply:
I was doing code cleanup, but I agree this might not be correct, so removed the code cleanup.